lib/services/devices/deviceRegistryMongoDB.js

```diff
@@ -61,7 +61,7 @@ function storeDevice(newDevice, callback) {
     var deviceObj = new Device.model(),
         attributeList = ['id', 'type', 'name', 'service', 'subservice', 'lazy', 'commands', 'staticAttributes',
             'active', 'registrationId', 'internalId', 'internalAttributes', 'resource', 'apikey', 'protocol',
-            'endpoint', 'transport', 'polling', 'timestamp', 'autoprovision'];
+            'endpoint', 'transport', 'polling', 'timestamp', 'autoprovision','explicitAttrs'];
```

Comment:
```suggestion
            'endpoint', 'transport', 'polling', 'timestamp', 'autoprovision', 'explicitAttrs'];
```

Just a small coding style fix ;)

Reply:
Fixed in 051c7d9